Vulnerabilities/stealing-tokens.md

```diff
@@ -0,0 +1,39 @@
+# CTI-2023-ADA-1103: Stealing validity tokens
+
+## Author
+
+Vacuumlabs -- Smart contract auditing team
+
+### Summary
```

Comment:
As "Summary" and "Description" are words that are very close, I suggest renaming the former to "Short Summary".